doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_debuggingr.rst

```diff
@@ -1,5 +1,3 @@
-:orphan:
-
 .. _ug_nrf54h20_debugging:
```

Comment:
@nordic-krch Hey Krzysztof, could you review this page?

Reply:
@ns-tolu Hey Tomasz, could you check if this document align with the most recent notes on lilium?